src/front/app/features/auth/components/LoginButton.scss

```diff
@@ -0,0 +1,15 @@
+.login-button {
+  display: block;
+  margin-top: 30px;
```

Comment:
Let's use rem values instead px ones in styles for easy optimization to different screen sizes. You can use th-rem() function from common/mixing.scss